### ISSUE TITLE
[Beta bug] Left-hand submenu doesn't expand when visiting a top level section directly

### ISSUE THREAD

**Issue by andrew** [ipfs/docs#425]
When directly visiting a top level section (like [Community](https://docs-beta.ipfs.io/community/)) on the new docs beta (i.e. clicking from offsite or pasting https://docs-beta.ipfs.io/community/ directly into your address bar), the left hand menu doesn't expand for the highlighted section:

![Screenshot 2020-01-21 at 14 07 29](https://user-images.githubusercontent.com/1060/72811355-63a80b00-3c57-11ea-8d71-aa8af6f2751f.png)

I would expect it to be expanded in the same was as if I clicked on the "Community" link in the menu:

![Screenshot 2020-01-21 at 14 08 20](https://user-images.githubusercontent.com/1060/72811396-7f131600-3c57-11ea-8098-9ef44a2a501b.png)

I happens for all the top level sections, not just Community, but it does work as expected for sub-sections like [Community >> IRC](https://docs-beta.ipfs.io/community/irc/)

**Comment by cwaring** [comment]
Good catch, I'll try to patch this up in the next round of menu refactors!

**Comment by jessicaschilling** [comment]
Also for context, see https://github.com/ipfs/docs/issues/389 -- we'll be making future changes to the nav overall that, while they don't address what you've brought up in this issue, do speak to overall nav evolution. Ideally we'd get this submenu concern addressed either before, or in parallel, with the work that closes #389.
